be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/SocialMediaValidator.scala

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.pop.pubsub.graph.validators
+
+import ch.epfl.pop.model.network.JsonRpcRequest
```

Comment:
Does not look very clean. A lot of artifacts in this file. Do you still need all the commented code?

Reply:
nope, removing useless stuff